oh_my_minishell/execute_pwd.c

```diff
@@ -0,0 +1,28 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   execute_pwd.c                                      :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: jikang <jikang@student.42seoul.kr>         +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2021/01/04 21:00:50 by jikang            #+#    #+#             */
+/*   Updated: 2021/01/06 19:28:43 by jikang           ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "minishell.h"
+
+/*
+** https://man7.org/linux/man-pages/man2/getcwd.2.html
+** char *getcwd(char *buf, size_t size);
+** getcwd 는 buf 가 NULL 이면, 동적할당을 해준다. 그래서 free 필요.
+*/
+
+void execute_pwd(void)
+{
+	char *pwd;
+
+	pwd = getcwd(NULL, 1024);
+	ft_putendl_fd(pwd, 1);
```

Comment:
```suggestion
	else
		ft_putendl_fd(pwd, 1);
````
위의 조건에서 바로 리턴되는게 아니라 else에 넣어줘야할 듯

Reply:
다음 커밋에서 수정 하였습니다, 감사합니다~